stable/clamav/values.yaml

```diff
@@ -29,22 +32,33 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+# Use with DaemonSet enable!
```

Comment:
Can you move this to deamonset so it will something like:

```
# Enable deamonset to scan the path / on every node (uncomment limitations to exlude master nodes).
deamonset:
   enable: false
   clamPathConfig:
     volumeMounts:
     - name: data-vol
       mountPath: /data
     - name: host-fs
       mountPath: /host-fs
       readOnly: true
     - name: logs
       mountPath: /logs
```